src/shapes/polyline.class.ts

```diff
@@ -118,7 +126,7 @@
      */
     _calcDimensions: function() {
 
-      var points = this.exactBoundingBox ? this._projectStrokeOnPoints() : this.points,
+      var points = this.exactBoundingBox ? this._projectStrokeOnPoints().map(elem => elem.projectedPoint) : this.points,
```

Comment:
So now after your elegant fix is there a need to return anything else than the projected points?